utils/run_test.sh

```diff
@@ -52,6 +52,26 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES="${DAOS_BASE}/test_results/*.xml"
+    for f in $FILES
+    do
+        if [ -f "$f" ]; then
+            echo "Processing XML $f"
+
+            SUITE=`grep "testsuite name=" "$f" | \
+                   grep -Po "name=\"\K.*(?=\" time=)"`
```

Comment:
As shellcheck has been saying, use $(...) here instead of ``.  The latter is unnestable and can be difficult to see in some fonts.

Reply:
ok. will change